qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/AllergiesSteps.java

```diff
@@ -65,4 +82,71 @@ public void removeNoKnownAllergy() {
 	public void systemRemovesNoKnownAllergies() {
 		assertTrue(textExists("Unknown"));
 	}
+	
+	@When("a user clicks Add New Allergy button")
+	public void loadAddAllergyPage() {
+		addOrEditAllergyPage = allergyPage.clickOnAddNewAllergy();
+	}
+	
+	@Then("the system loads add new allergy page")
+	public void systemLoadsAddAllergyPage() {
+		assertTrue(textExists("Add New Allergy"));
+	}
+	
+	@And("a user selects an allergy")
+	public void selectKnownAllergy() {
+		addOrEditAllergyPage.enterDrug(DRUG_NAME);
+		addOrEditAllergyPage.drugId();
+		addOrEditAllergyPage.enterReaction(REACTION);
+		addOrEditAllergyPage.reactionId();
+		addOrEditAllergyPage.addAllergyNote(ALLERGY_NOTE);
+	}
+	
+	@And("a user clicks on the save allergy button")
+	public void saveAllergy() {
+		addOrEditAllergyPage.clickOnSaveAllergy();
+	}
+	
+	@Then("the system adds known allergies into allergies table")
```

Comment:
can the system save(add) more than one allergy at the same time?

Reply:
> can the system save(add) more than one allergy at the same time?

no